bor/keeper_test.go

```diff
@@ -0,0 +1,69 @@
+package bor_test
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/magiconair/properties/assert"
```

Comment:
Let's use assert from `stretchr` package which is commonly used by other test cases

Reply:
removed